dask_cuda/dask_cuda_worker.py

```diff
@@ -252,6 +264,7 @@ def del_pid_file():
             contact_address=None,
             env={"CUDA_VISIBLE_DEVICES": cuda_visible_devices(i)},
             name=name if nprocs == 1 or not name else name + "-" + str(i),
+            worker_class=CUDAWorker,
```

Comment:
So previously this was `Nanny`, which created a new process that started a `Worker`.  Now we have `CUDAWorker`, which has the nice CUDA device memory logic, but doesn't start a new process.  I wonder if maybe we need something like 

```python
class CUDANanny(Nanny):
    Worker = CUDAWorker
```

(or some other similar solution)

This is, I think, what is causing some of the test failures in `dask_cuda/tests/test_local_cuda_cluster.py`

Reply:
In particular, this is what is causing the errors with the `env=` keyword not being available.  We actually need to use a Nanny here so that we can set `CUDA_VISIBLE_DEVICES` as an environment variable before starting a worker.

Reply:
No, `worker_class` was `Worker`, but `t` is `Nanny`. See https://github.com/rapidsai/dask-cuda/blob/8659fb6a01bd532663d3a1cffb67ffb8254b013f/dask_cuda/dask_cuda_worker.py#L225 and https://github.com/rapidsai/dask-cuda/blob/8659fb6a01bd532663d3a1cffb67ffb8254b013f/dask_cuda/dask_cuda_worker.py#L249.

Reply:
I see what you mean now. You probably only tagged the wrong piece of code, I think you meant to do it in https://github.com/rapidsai/dask-cuda/pull/35/files#diff-adfdb0f8dc34800895529621f06b1340R60 instead. Thanks for pointing out what's the issue, I'll fix it.